test/e2e/lib/get_account.ts

```diff
@@ -0,0 +1,24 @@
+import { TestAccount, TestAccountName } from '@automattic/calypso-e2e';
```

Comment:
Nitpick: regarding filenames in the Calypso repo, we generally use hyphens to separate words in filenames. Yes there are some special cases where we might use underscores for namespacing, but generally it's kebab case.

Same thing for the `pw_base.ts` file which was added in the last PR.

Reply:
sorry about that, updated them now

Reply:
for the specs files, I used existing names and added `.spec` towards the end, are these still ok?

eg. `signup__smoke.spec.ts`

Reply:
Yes, I think the `__` for namespacing is ok.

Reply:
And the `.spec` too 😁